### ISSUE TITLE
R4: How to build an application ontology with Robot

### ISSUE THREAD

**Issue by nsjuty** [FAIRplus/FAIRPlus_squad1#8]
Fuqi, Dani & Emiliano

doc:

Tasks in building and maintaining an application ontology include,generating subsets, QC, generating release files, etc. ROBOT provides many command covering these tasks. An example showing how to build an application ontology would be nice. 
1. What are the tasks for building and maintaining ontologies.
2. how to use ROBOT for these tasks
3. provide an example

**Comment by FuqiX** [comment]
Detailed steps from @daniwelter  
1. Identify appropriate source ontologies
2. Build the top level "umbrella" ontology
3. Extract ontology modules from source ontologies
4. Module extraction - using text lists
5. Module extraction - using SPARQL
6. Merge extracted modules under the umbrella
7. Post-merge modifications
@ereynrs Please see if there is anything to add.

**Comment by emilianoreynares** [comment]
I agree with the @daniwelter 's proposal as a first recipe draft. New stages and sub-stages will raise through detailing these general steps and developing the specific use case.
E.g, Would be interesting to test (a) the ROBOT functionalities aimed to reason over the built ontology,  (b) those aimed to verify and validate it, and (c) its versioniong (diff/compare) capabilities.
